index.html

```diff
@@ -1,34 +1,81 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+
  <head> 
-    <title>Raissa's</title>
+
+    <meta charset="UTF-8">
+    <title>My First Webpage</title>
+    
 </head>
 
  <body>
-
-   <h1>"About" </h1>
+   <header>
+   <h1>Raíssa Gomes </h1>
+   
+   <nav>
+     <ul>
+       <li><a href="#">Home</a></li> 
+       <li><a href="#about">About</a></li>
+       <li><a href="#experience">Experience</a></li>
+       <li><a href="#connect">Connect</a></li>
+     </ul>
+  </nav>
  
-   <p>Hi, My name is  Raíssa, I live in Chicago, and I'm currently a student at Code the Dream, I am learning HTML, CSS and JavaScript. 
+   </header>
 
-   </p>  
+<section id="about">
 
-<h2>"Experience"</h2>
+     <h2>About</h2>
+     <p>Hi, My name is  Raíssa, I live in Chicago, and I'm currently a student at Code the Dream, I am learning HTML, CSS and JavaScript. </p>  
 
-<ul>
-<li>HumanPredictions</li> 
-<li>Notre-Shop</li>
-<li>Fishmen Fabrics</li>
-</ul>
+</section>
 
-<h3>"Connect"</h3>
+<section id="experience">
+    <h2>Experience</h2>
 
-<ul>
-    <li>raissanfg@gmail.com</li>
-    <li>https://twitter.com/raissanfg</li>
-    <li>https://www.instagram.com/raissanfg</li>
-</ul>
+       <ul>
+         <li>HumanPredictions</li> 
+         <li>Notre-Shop</li>
+         <li>Fishmen Fabrics</li>
+       </ul>
 
+</section>
 
- </body>
 
+<section id="skills">
+   <h2>Skills</h2>
+   <ul class="skillSection"></ul>
+
+     <table border="3">
+       <tr>
+         <th>HTML</th>
+         <th>CSS</th>
+         <th>JavaScript</th>
+
+       </tr>
+     </table>
+
+</section>
+
+
+<section id="connect">
+    <h2>Connect</h2>
+
+      <ul>
+         <li>raissanfg@gmail.com</li>
+         <li>https://twitter.com/raissanfg</li>
+         <li>https://www.instagram.com/raissanfg</li>
+      </ul>
+
+</section>
+
+<footer>
+    <p>&copy;2022 Raissa Gomes</p>
```

Comment:
Once you have your script working; you should remove this part; since you should be adding it from the script for this assignment